spec/systems/users_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :system do
+  before do
+    @user = User.find(1)
+  end
+  describe 'index action page' do
```

Comment:
![image](https://github.com/mahammad-mostafa/topics-blog/assets/93039359/c48a0b53-efc8-4c21-9b47-7427562aca2d)

- [ ] Great work so far on making the tests but the tests are failing on my side because your tests are dependent on the entries in the database of the app which is alarming because tests should not be dependent on the app 🤔 It would be great if you create entries here in the before block to make your tests independent of the app database 👍 These errors come when your database has not entries 🤔 You can empty your database and fix these errors 😊